src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -2267,6 +2270,50 @@ export class ResultsViewPageStore {
         }
     });
 
+    readonly referenceGenes = remoteData<ReferenceGenomeGene[]>({
+        await: ()=>[
+            this.studies,
+            this.mutationEnrichmentData,
+            this.copyNumberAmpEnrichmentData,
+            this.genes
+        ],
+        invoke: async () => {
+            const queryGenes = this.genes.result!.map((g:Gene)=>g.hugoGeneSymbol.toUpperCase());
+            const mutGenes = this.mutationEnrichmentData.result!.map(
+                                    (a:AlterationEnrichment)=>a.hugoGeneSymbol.toUpperCase());
+            const cnvGenes = this.copyNumberAmpEnrichmentData.result!.map(
+                                        (a:AlterationEnrichment)=>a.hugoGeneSymbol.toUpperCase());
+            return fetchReferenceGenomeGenes(this.studies.result[0].referenceGenome,
+                                             this.hugoGeneSymbols.concat(queryGenes, mutGenes, cnvGenes));
+        }
+    });
+
+    @computed get hugoGeneSymbolToCytoband() {
```

Comment:
(1) After discussion over time, @alisman and I have determined that in cases like this (taking this as an example): it should be a remoteData which awaits this.referenceGenes, rather than a computed which checks the referenceGenes status.
(2) Isn't the name unclear as written, because it does not distinguish from the cytoband of a gene not on the selected reference genome? I think it should have the word "reference gene" in there somewhere
(3) `hugoGeneSymbolToCytoband` and `hugoGeneSymbolToChromosome` are redundant - should just be `hugoGeneSymbolToReferenceGene`, then users can do `hugo...Gene[gene].cytoband` or `.chromosome`

`referenceGeneHugoGeneSymbolToCytoband` is better, to distinguish from the query genes that are not from the selected reference genome. Also, please make this a `remoteData` which awaits `this.referenceGenes`. Please also make `hugoGeneSymbolToChromosome`